phpstan-baseline.neon

```diff
@@ -10199,6 +10199,16 @@ parameters:
 			count: 2
 			path: modules/social_features/social_group/modules/social_group_invite/social_group_invite.module
 
+		-
```

Comment:
Did you try to resolve these phpstan issues in code? We should try to avoid adding such exceptions in the baseline.

Reply:
@bojanbogdanovic Resolved ;-)